app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  @@renderer = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: true)
```

Comment:
@itsWill FYI: I bumped into [that old blog post](http://snippets.aktagon.com/snippets/299-class-variables-vs-class-instance-variables-in-ruby) today. It's a simple code snippet, but I think it drives home the distinction between *class variables* and *class instance variables*, which is related to our previous discussion.

Note that Hound CI made me modify that Helper code to use a *class instance variable* instead of a *class variable*. Not a bad recommendation, and nor does it undermine the aforementioned performance and memory efficiency benefits.